scripts/build/common.rollup.js

```diff
@@ -28,11 +26,6 @@ module.exports.commonPlugins = [
     include: '/**/*.scss',
     includePaths: [path.resolve('node_modules')],
     prefix: globalSassImports,
-    processor: css =>
-      postcss({
-        plugins: [autoprefixer({ grid: 'autoplace' })],
-      })
-        .process(css, { from: undefined })
-        .then(result => result.css),
+    processor: css => css,
```

Comment:
Question: do we still need a processor if it only does the identity transform?

Reply:
that a good catch, we don't need it!